### PR TITLE
Updated ASM8 to ASM9

### PR DESCRIPTION
**Pull request by mstahv**

### CODE REVIEW

File: flow-server/src/main/java/com/vaadin/flow/server/frontend/scanner/FrontendAnnotatedClassVisitor.java [hunk 1]
```diff
@@ -60,7 +60,7 @@
      *            The annotation class name to visit
      */
     FrontendAnnotatedClassVisitor(ClassFinder finder, String annotationName) {
-        super(Opcodes.ASM8);
```
Comment (knoobie) [inline flow-server/src/main/java/com/vaadin/flow/server/frontend/scanner/FrontendAnnotatedClassVisitor.java:63]: This (https://gitlab.ow2.org/asm/asm/-/issues/317854) could probably explain why the OpCodes has to be updated for newer java releases to work properly. Comment by Remi Forax: 

> I will close this issue as work as intended even if you may think it's surprising, currently ASM ClassReader support Java 11 (and even Java 12) so it recognizes Java 11 new attributes (here NestMembers). But here you are using a ClassVisitor which is configured to only recognize the ASM6 API, thus it fails at runtime with an UnsupportedOperation exception.

**Comment by Artur-** [comment]
Can we merge this? Why is it a draft?

**Review by Artur- — APPROVED** [review]

**Comment by mstahv** [comment]
I can't test that it actually works. I cannot reproduce the issue. Maybe going in blind is sometimes the option 😎

**Comment by mstahv** [comment]
I can't test that it actually works. I cannot reproduce the issue. Maybe going in blind is sometimes the option 😎 @vaadin/flow Up to you to make the call.

**Review by caalador — APPROVED** [review]

**Comment by mstahv** [comment]
@caalador Can you CP this for Vaadin 14 series as well?

**Comment by mstahv** [comment]
@caalador For some reason the fix has not been picked to 9.0 (aka Vaadin 22) branch. https://github.com/vaadin/flow/blob/9.0/flow-server/src/main/java/com/vaadin/flow/server/frontend/scanner/FrontendAnnotatedClassVisitor.java#L63

**Comment by caalador** [comment]
Auto merge had not merged for some reason.

**Pull request merged**